mongodb/src/main/java/org/hibernate/ogm/datastore/mongodb/query/parsing/nativequery/impl/NativeQueryParser.java

```diff
@@ -76,6 +80,14 @@ public Rule Separator() {
 		return Sequence( ZeroOrMore( WhiteSpace() ), ". " );
 	}
 
+	public Rule Reserved() {
+		return FirstOf(
+				Sequence( Find(), builder.setOperation( Operation.FIND ) ),
```

Comment:
`builder.setOperation()` is already invoked by `Operation()`. I don't think we need to do it here, too?